hello/lib/hello_web/views/layout_view.ex

```diff
@@ -1,6 +1,10 @@
 defmodule HelloWeb.LayoutView do
   use HelloWeb, :view
 
+
+  def title() do
```

Comment:
@Vretamoza by coding style functions with arity: 0 should not have parentheses

Reply:
I changed it already